presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -1304,6 +1304,96 @@ public void testJoinPredicateMoveAround()
     }
 
     @Test
+    public void testSimpleFullJoin()
+            throws Exception
+    {
+        assertQuery("SELECT a, b FROM (VALUES (1), (2)) t(a) FULL OUTER JOIN (VALUES (1), (3)) u(b) ON a = b",
+                "SELECT * FROM (VALUES (1, 1), (2, NULL), (NULL, 3))");
+        assertQuery("SELECT COUNT(*) FROM lineitem FULL JOIN orders ON lineitem.orderkey = orders.orderkey",
```

Comment:
This test will behave like an inner join since every `orderkey` is in both tables. Copy them and use this for the join to produce non-matching on both sides: `orders.custkey = lineitem.orderkey`